challenges/objects-arrays.js

```diff
@@ -45,21 +69,21 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+const universities = graduates.map(graduate => graduate.university);
```

Comment:
You're _not_ allowed to use array methods here! Make sure you read all the instructions first, before heading straight into solving the challenge! Read line 69 above, `Request 1` indicated that no array methods are allowed to solve this problem.